HTMLSerializer.js

```diff
@@ -122,8 +160,8 @@ var HTMLSerializer = class {
     var win = element.ownerDocument.defaultView;
     var style = win.getComputedStyle(element, null).cssText;
     var windowDepth = this.windowDepth(win);
-    style = style.replace(/"/g, this.escapedQuote(windowDepth+1));
-    var quote = this.escapedQuote(windowDepth);
+    style = style.replace(/"/g, this.escapedCharacter('"', windowDepth+1));
```

Comment:
I agree with you that this part is confusing.  Are you suggesting that I simply change the variable name from windowDepth to nestingDepth? I think that would be a better choice of variable name, but I still think it would be a little confusing.  Do you think it's clear why the contents of the attribute have to be escaped an extra time?